app/src/main/java/org/mozilla/fenix/settings/autofill/AutofillSettingFragment.kt

```diff
@@ -94,6 +94,11 @@ class AutofillSettingFragment : BiometricPromptPreferenceFragment() {
             isChecked = context.settings().shouldAutofillCreditCardDetails
             onPreferenceChangeListener = SharedPreferenceUpdater()
         }
+
+        requirePreference<SwitchPreference>(R.string.pref_key_addresses_save_and_autofill_addresses).apply {
+            isChecked = context.settings().shouldAutofillAddressDetails
```

Comment:
should we also hide this depending on the flag?

Reply:
I think it's fine as is. iirc we don't show this fragment unless the flag is enabled anyways.